### ISSUE TITLE
Ideas for improvement Gantry

### ISSUE THREAD

**Issue by kosh2323** [gantry/gantry5#3041]
Hello everyone. The thread will be dedicated to ideas for improving the user experience. I suggest that everyone who wants to write ideas that are not very difficult from the point of view of implementation, so that it is easier for developers. And so the first thing I would like to see in the future: 

1. Dragging sections, in some frameworks this is implemented (Helix, Astroid). After all, particles can be dragged with the mouse, but sections cannot. For example, the Outline template is not suitable for everyone, because there are 3 columns in the main container, and navigation is higher than header, if you need to do it differently, then only through Custom. 

2. It would be nice to add an adaptivity class field to all particles, make a drop-down list, and register classes from the documentation in the values, because they work for display on mobile devices.

https://docs.gantry.org/gantry5/tutorials/utility-classes

**Comment by mahagr** [comment]
Sections in Grav are supposed to be set by the theme creator. They are defined in a preset layout file.

**Comment by kosh2323** [comment]
> Sections in Grav are supposed to be set by the theme creator. They are defined in a preset layout file.

Yes, I understand, but it's about joomla, these are wishes from the point of view of user convenience. If it were, then the task of making your own template files for your theme disappears, everything would be dynamic.

**Comment by mahagr** [comment]
Sorry, I meant Gantry, not Grav. :)

**Comment by kosh2323** [comment]
Aaaa...I understood, anyway, I understand that there is not enough time, so first of all I have to correct mistakes, and suggestions for improvement go by the wayside.....
